data/json/monsters/nether.json

```diff
@@ -1428,7 +1428,26 @@
     "families": [ "prof_wp_netherium_abomination" ],
     "weakpoint_sets": [ "wps_netherium_abomination" ],
     "harvest": "meatslug",
-    "special_attacks": [ [ "GENE_STING", 20 ] ],
+    "special_attacks": [
+      {
+        "type": "monster_attack",
+        "attack_type": "melee",
```

Comment:
are you sure it is melee? its a spell, it has different syntax

Reply:
I stole this spell
```
{
        "type": "monster_attack",
        "attack_type": "melee",
        "id": "mon_orc_mage_blood_bleeding_spell",
        "cooldown": 10,
        "move_cost": 100,
        "range": 8,
        "damage_max_instance": [ { "damage_type": "necrotic", "amount": 10 } ],
        "effects": [ { "id": "bleed", "duration": 60, "chance": 33, "intensity": 3, "affect_hit_bp": true } ],
        "dodgeable": true,
        "blockable": false,
        "hit_dmg_u": "%1$s growls words of magic and your %2$s is impacted by a bolt of carmine energy!",
        "hit_dmg_npc": "%1$s growls words of magic and <npcname>'s %2$s is impacted by a bolt of carmine energy!",
        "miss_msg_u": "%1$s growls words of magic and you narrowly avoid a bolt of carmine energy!",
        "miss_msg_npc": "%1$s growls words of magic and <npcname> narrowly avoids a bolt of carmine energy!",
        "no_dmg_msg_u": "%1$s growls words of magic and your %2$s is surrounded by a cloud of scratching thorns, but they do no damage.",
        "no_dmg_msg_npc": "%1$s growls words of magic and <npcname>'s %2$s is surrounded by a cloud of scratching thorns, but they do no damage."
      },
```
Because it looked like it would do all the things I wanted.

Reply:
But I think it's name as a spell is a lie lol.